src/services/wallets/metamask/metamask.ts

```diff
@@ -0,0 +1,20 @@
+import Web3 from "web3";
+import { HexString } from "../../../utilities/types";
+import ethereum, { EthereumProvider } from "./ethereum";
+
+const mmweb3 = new Web3(ethereum as EthereumProvider);
+
+export const getWeb3 = (): Web3 => {
+  if (!ethereum) throw new Error("Metamask not installed");
```

Comment:
Should this check `isInstalled()` instead? `ethereum` can be more than just metamask

Reply:
I'll go ahead and use isInstalled for now, but are there other wallets that may use the EthereumProvider?